.chromatic/config.js

```diff
@@ -1,4 +1,4 @@
-import 'storybook-chromatic';
```

Comment:
it's interesting, we don't actually have 'storybook-chromatic' as a dependency, it's being included because of 'react-inspector' which i also can't find anywhere
https://github.com/adobe/react-spectrum/blob/main/yarn.lock#L18870
I think it's just caught up in our yarn lock, I'll worry about that later, thanks for fixing this